examples/NavigationPlayground/js/App.js

```diff
@@ -103,7 +103,13 @@ const AppNavigator = StackNavigator({
   mode: Platform.OS === 'ios' ? 'modal' : 'card',
 });
 
-export default () => <AppNavigator />;
+export default () => (
+  <AppNavigator
+    screenProps={{
+      exampleScreenProp: 'exampleScreenPropValue',
```

Comment:
This is the top-level file for all examples, and I would prefer if it didn't get too bloated with random example stuff. Can we remove this example from `App` and `StacksInTabs`?

Perhaps we can create a new example for this that demonstrates how props can be deeply passed.